portfolio.js

```diff
@@ -0,0 +1,149 @@
+    projectsrc: '#',
+    githubsrc: '#',
+  },
+  {
+    id: 3,
+    projectname: 'Tonic',
+    subprojectname: 'canopy',
+    projecttype: 'Back end dev',
+    year: 2015,
+    image: 'images/portoflioCard/professional.png',
+    details:
+      "technology: ['html', 'css', 'javascript'],Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent.",
+    usedtechnology1: 'html',
+    usedtechnology2: 'css',
+    usedtechnology3: 'javascript',
+    projectsrc: '#',
+    githubsrc: '#',
+  },
+  {
+    id: 4,
+    projectname: 'Multi-Post Stories',
+    subprojectname: 'canopy',
+    projecttype: 'Back end dev',
+    year: 2015,
+    image: 'images/portoflioCard/uber.png',
+    details:
+      "Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent.",
+    usedtechnology1: 'html',
+    usedtechnology2: 'css',
+    usedtechnology3: 'javascript',
+    projectsrc: '#',
+    githubsrc: '#',
+  },
+];
+
+const hamburger = document.querySelector('#hamburger');
+const closed = document.querySelector('.close');
+const navListItems = document.querySelectorAll('.nav-list-item');
+const navList = document.querySelector('#nav-list');
+
+const mobileMenu = () => {
+  navList.classList.toggle('active');
+  hamburger.classList.toggle('close');
+  closed.classList.toggle('displayshow');
+};
+
+navListItems.forEach((element) => {
+  element.addEventListener('click', mobileMenu);
+});
+
+hamburger.addEventListener('click', mobileMenu);
+closed.addEventListener('click', mobileMenu);
+
+const images = document.getElementById('project-img');
+const details = document.getElementById('newdetail');
+const projectsrcs = document.getElementById('projectsrc');
+const projectnames = document.getElementById('projectname');
+const subprojectnames = document.getElementById('subprojectname');
+const years = document.getElementById('year');
+const frontendBackendDev = document.getElementById('frontend/backend');
+const technology1 = document.getElementById('tech1');
+const technology2 = document.getElementById('tech2');
+const technology3 = document.getElementById('tech3');
+const githubsrcs = document.getElementById('githubsrc');
+const button1 = document.getElementById('seeproject1');
+const button2 = document.getElementById('seeproject2');
+const button3 = document.getElementById('seeproject3');
+const button4 = document.getElementById('seeproject4');
+const closeBtn = document.querySelector('.popUpClose');
+const modal = document.querySelector('.modal-overlay');
```

Comment:
For a consistent code-base, and to avoid future problems it is advised to use `querySelectorAll` for querying classes and `querySelector` for IDs...

if `popUpClose` is used in many elements then this should be a `querySelectorAll` if it is only being used in one element, make it and ID instead

Reply:
Okay, I will make correction to this. popUpClose is used only in one element.